tools/doc/html.js

```diff
@@ -307,14 +329,19 @@ function buildToc(lexed, filename, cb) {
     depth = tok.depth;
     const realFilename = path.basename(realFilenames[0], '.md');
     const id = getId(realFilename + '_' + tok.text.trim());
+
     toc.push(new Array((depth - 1) * 2 + 1).join(' ') +
-             '* <a href="#' + id + '">' +
-             tok.text + '</a>');
+             '* %stability_' + tok.stability + '%' +
```

Comment:
@ilfroloff Would this be better as
```js
var toc_entry = tok.stability ? `<li class="${tok.stability}">` : '<li>';
toc_entry += `<a href="#' + id + '">${tok.text}</a>`;

toc.push(new Array((depth - 1) * 2 + 1).join(' ') + toc_entry);
```

That would get rid of the massive find/replace after the fact?